docs/manual/developer_guide.adoc

```diff
@@ -536,6 +536,18 @@ benchmark_root: "../linux_os/guide"
 .....
 ----
 
+Rules from multiple locations can be used for a single Benchmark. There is an optional key `extra_rule_group_roots` for a list of paths to some arbitrary Groups of Rules to be processed.
```

Comment:
We need to say here how it works with inheritance. For example, the `platform: machine` can be specified both in the rule and in one of parent groups of the rule. Obviously, one can select a group that is not explicit on platform, because it is inherited from its parent group, or I can select a group that defines platform explicitly. How will those rules end up?

Reply:
Right now it works pretty much like if you just symlink this external group directory into the benchmark. Not sure how it should work, thought.

Reply:
If this behaviour is what we would like from this improvement I can add there something along the lines that rules would be processed as if they were located inside the directory structure of the benchmark in question and/or relate to a section in the manual that describes how rules are processed in general.

Reply:
I have checked, and I agree with your comments. The only case to watch out for is when you specify a directory of a group that normally inherits properties from its parent group. I have verified that it indeed works as if you symlink the directory into the main benchmark - the target group is disconnected from its parent.
Next, please add a note that if you include an additional directory to the benchmark, following rules will become part of the benchmark:

- rules that have the corresponding prodtype specified
- rules that have no prodtype metadata